hieradata/type/desktop.yaml

```diff
@@ -7,4 +7,7 @@ ocf::auth::gsudo: [ocfstaff]
 # Don't manage staff home directories on desktops.
 ocf::staff_users::noop: true
 
+# Allow ocfroot to use docker without sudo
+ocf::packages::docker::admin_group: ocfroot
```

Comment:
from my understanding, allowing all `ocfstaff` may be a security concern?

Reply:
docker == root

if ocfstaff has root on desktops, it's okay to give them docker
if only ocfroot has root on desktops, we should probably keep docker to ocfroot only

Reply:
ocfstaff has root on desktops, but most staff can't use it because they don't have /root principals.